jax/experimental/export/_export.py

```diff
@@ -307,6 +307,17 @@ def __str__(self):
     # do not want the entire serialized module to end up in locations.
     return f"Exported(fun_name={self.fun_name}, ...)"
 
+  def in_shardings_for_mesh(self,
```

Comment:
rename to `get_named_shardings` or something similar?

Reply:
I changed it to `xla_compatible_in_shardings`.